docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+# Tizen 5.0 Public M2 Release Notes
+
+Release date: Oct. 31, 2018
+
+
+## System (Kernel and System framework)
+
+### New and changed features
+
+- New APIs and features
+  - D-Bus
+    - A policy rule checker has been developed and deployed for improving security.
+  - Dlog
+    - A new feature of front-end filters has been developed for efficient logging and facilitating runtime log analysis.
+  - Deviced
+    - C# APIs for display control have been added.
+    - Native APIs for multi-LED control have been added.
+  - Storaged
+    - Native and C# APIs for storage device types have been added.
+    - A new storage cleanup policy has been developed.
+  - Compressed fs
+    - Btrfs, Squashfs, and Overlayfs have been supported for rootfs.

```

Comment:
According to your latest comment, we will keep "present perfect"(have been/has been).
Rephrase as:
Btrfs, Squashfs, and Overlayfs have been supported for rootfs.

Reply:
GLOBAL COMMENT:
@wtprince 
Yes.
To maintain consistency with other release notes, we will keep "present perfect"(have been/has been).
Please keep the sentences with"has been or have been" according to the sentence.
Both "has been” and “have been” both are used in present perfect continuous.
“Has been” is more commonly used for third person , while “have been” can be used for both first person and second person.It can also be used as a plural form for third person.

Reply:
It's OK.